src/Shell_Command.php

```diff
@@ -76,7 +76,7 @@ public function __invoke( $args, $assoc_args ) {
 		$shell   = ( 'mailhog' === $service ) ? 'sh' : 'bash';
 		$command = \EE\Utils\get_flag_value( $assoc_args, 'command' );
 		if ( $command ) {
-			EE::exec( "docker-compose exec $user_string $service $shell -c \"$command\"", true, true );
+			EE::exec( "docker-compose exec -T $user_string $service $shell -c \"$command\"", true, true );
```

Comment:
Don't directly do `-T` here. By not allocating a tty the output generated is not pretty.

Here is the difference:

Without `-T`:
```bash
$ docker-compose exec php bash -c 'wp plugin list'   
+-----------------------------+----------+--------+---------+
| name                        | status   | update | version |
+-----------------------------+----------+--------+---------+
| akismet                     | inactive | none   | 4.1     |
| hello                       | inactive | none   | 1.7.1   |
+-----------------------------+----------+--------+---------+
```

Same output with `-T`:
```bash
$ docker-compose exec -T php bash -c 'wp plugin list'
name	status	update	version
akismet	inactive	none	4.1
hello	inactive	none	1.7.1
```

As not allocating a tty is need for specific commands when using the command through ssh as mentioned in the [issue](https://github.com/EasyEngine/shell-command/issues/16). So, please add `-T` on specific flag invocation like when say `--skip-tty` is passed. Not by default as in this case.